### DISCUSSION TITLE
Is it possible to toggle a breakpoint in a jdt:// viewed file of a third party library ?

### DISCUSSION THREAD

**Discussion by darengrant** [mfussenegger/nvim-jdtls#323] [Q&A] [answered]
I have jdtls all working well on a multi-module gradle project and am able debug my own code and step into third party code without issue.  But I do not appear to be able to toggle a new breakpoint in any third party code opened via a jdt:// based link.

My question is am I doing something wrong, or is this capability not presently available in nvim-jdtls ?

**Comment by mfussenegger** [comment, marked as answer]
Should work:


https://user-images.githubusercontent.com/38700/190710024-e84a29d9-f596-4ae2-aa11-dc5d4953f6e9.mp4

**Reply by darengrant** [reply]
Thank you for confirming that it is possible.

It is not working for me, It must be an issue with my setup, but at least now I know that all I need to do is debug my setup to get it working.

Thanks again
